zdm/parameters.py

```diff
@@ -228,16 +227,24 @@ class IGMParams(data_class.myDataClass):
 # FRB intrinsic width parameters
 @dataclass
 class WidthParams(data_class.myDataClass):
+    WidthFunction: int = field(
+        default=2,
+        metadata={
+            "help": "ID of function to describe width distribution. 0: log-constant, 1:log-normal, 2: half-lognormal",
+            "unit": "",
+            "Notation": "",
+        },
+    )
     Wlogmean: float = field(
-        default=1.70267,
+        default=-0.29,
```

Comment:
where do these defaults come from?  Maybe add a comment or even something to the metadata.